packages/db-migrations/databases/fxa/patches/patch-139-140.sql

```diff
@@ -0,0 +1,14 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('139');
+
```

Comment:
This should have the truncate table before the alter statements

Reply:
I commented in teh jira: I think we shouldn't make the migration even longer by combining the data deletion and the alters.  SREs could do that ahead of the migration.  Or I can add another migration into the repo.

Reply:
So long as it gets run before the alters it doesn't matter to me.  Let's coordinate with SRE 👍

Reply:
Something we did in another project (possibly Monitor) was made SREs [CODEOWNERS](https://github.com/mozilla/fxa/blob/main/.github/CODEOWNERS) of the packages/db-migrations/** patches so they'd get notified and be optional reviewers of each DB related PR. Not sure if that'd be useful here or just excess noise for @jbuck @dlactin and @rachaelcrook